code/modules/projectiles/guns/projectile/revolver.dm

```diff
@@ -102,6 +103,7 @@
 	mag_type = /obj/item/ammo_box/magazine/internal/cylinder/rev38
 	unique_rename = 1
 	unique_reskin = 1
+	pin = /obj/item/device/firing_pin/generic
```

Comment:
The parent revolver was the syndicate variety available to nukeops, so I had to edit it to have the syndicate pin. It wouldn't make sense for the other ones to have syndicate pins too, so I just made them have generic ones.